grails-app/views/speciesListItem/list.gsp

```diff
@@ -1133,5 +1183,67 @@
         });
     });
 </asset:script>
+
+<script type="module">
+    import { createApp } from 'https://unpkg.com/petite-vue?module'
```

Comment:
This looks like a bit of boilerplate that has slipped through. Is it intentional?